src/main/java/nextstep/courses/domain/session/Schedule.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.NotEnrollmentPeriodException;
+
+import java.time.LocalDate;
+
+public class Schedule {
+
+    private final LocalDate startDate;
+
+    private final LocalDate endDate;
+
+    public Schedule() {
+        this(LocalDate.now(), LocalDate.now());
+    }
+
+    public Schedule(LocalDate startDate, LocalDate endDate) {
+        this.startDate = startDate;
+        this.endDate = endDate;
+    }
+
+
+    public final boolean isEnrollmentPeriod() {
+        return SessionStatus.status(this).equals(SessionStatus.ENROLLING);
```

Comment:
객체간의 관계가 복잡해 보여요.

```suggestion
        return !isBeforeStartDate(LocalDate.now()) && !isAfterEndDate(LocalDate.now());
```

🤔